### DISCUSSION TITLE
第2节课作业(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#255] [作业]
作业链接贴在这里！
笔记链接**不**贴在这里！

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/blob/main/Lecture2/InternLM_Demo%E5%B7%A5%E7%A8%8B%E7%AC%94%E8%AE%B0.pdf

**Reply by zcg-joker** [reply]
！！！

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/homework/lesson-2.md

**Reply by zcg-joker** [reply]
！！！

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson2/Lesson%202%20%E4%BD%9C%E4%B8%9A.md

**Reply by zcg-joker** [reply]
！！！

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135519880?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135519880%22%2C%22source%22%3A%22sxswz%22%7D

**Reply by zcg-joker** [reply]
！！！

**Comment by jimmyrich7203** [comment]
第二节作业 1609 
https://github.com/jimmyrich7203/InternLMlecture/blob/main/%E6%B5%A6%E8%AF%AD%20%E7%AC%AC%E4%BA%8C%E8%8A%82%E4%BD%9C%E4%B8%9A.pdf

**Reply by zcg-joker** [reply]
！！！

**Comment by SpongeBob-0715** [comment]
https://p065drvsp5m.feishu.cn/docx/Q8Yud8h3Vo66yLxuvO6cUnmsnDa?from=from_copylink

**Reply by zcg-joker** [reply]
！！！
做作业的过程还可以加入一点自己的思考哈~

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17957523

**Reply by zcg-joker** [reply]
！！！
进阶作业好像还少了浦语·灵笔的部分哈~

**Comment by qqqmmmxxx** [comment]
https://zqm6a0l4f0l.feishu.cn/wiki/LnlvwYQ8jiR4lck0fr4ccSyjn6c?from=from_copylink

**Reply by zcg-joker** [reply]
！！！

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson2_homework.md

**Reply by zcg-joker** [reply]
！！！珠海很美~

**Comment by jhyz** [comment]
https://blog.csdn.net/jinghangyz/article/details/135557578?spm=1001.2014.3001.5502

**Reply by zcg-joker** [reply]
！！！出现问题炸了，但解决了，真不错~

**Comment by fxlt** [comment]
https://github.com/fxlt/internLM-Study/blob/main/2.%E8%BD%BB%E6%9D%BE%E7%8E%A9%E8%BD%AC%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E5%A4%A7%E6%A8%A1%E5%9E%8Bdemo_%E4%BD%9C%E4%B8%9A.pdf

**Comment by 12345567890hc** [comment]
https://www.mubu.com/doc/5lOS4nmo-EC

**Reply by zcg-joker** [reply]
！！！后面两个作业再尝试一下哈~ 有问题可以在群里提出来~

**Comment by LordHidy** [comment]
https://github.com/LordHidy/InternLM_learning/blob/main/Assignments/Lesson_2_hw.md

**Reply by zcg-joker** [reply]
！！！好一个桃花侠大战菊花怪~

**Comment by Alphonsinaone** [comment]
第2节课作业
https://github.com/Alphonsinaone/InternLM/blob/main/Homework/Homework_Lesson_2.md

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135561653?spm=1001.2014.3001.5501

**Comment by IdealDestructor** [comment]
https://blog.deqiang.wang/2024/01/12/Project-LLM2/

**Comment by wunuooo** [comment]
https://blog.csdn.net/ynx005/article/details/135578166?spm=1001.2014.3001.5502

**Comment by wade361** [comment]
https://github.com/wade361/-Wade/issues/3
